### DISCUSSION TITLE
How to mine in pool to receive just some coins? As solo is dying too fast for small players.

### DISCUSSION THREAD

**Discussion by alexey-sl** [Chia-Network/chia-blockchain#4865] [Q&A]
I have a probem. 
For now I setup a dedicated linux server with full node and it is a fatmer. 
Behind this machine is 5 little linux conputers on net,as a harvesters and plot-builders. 
So I'm making a little chia cluster.
But I have weak computers. So they build only 2 plots a day each. 

When I started this project, was full of enthusiasm. 
And it passed like a week, but difficulty growth up drammatically. 
For the example - my friend never mind to that, and is mining on a pool(all known pool).
For this moment he receives money EVERY DAY for his investitions and doing NO effort at all. 
But I receive zero money and wasted a mass of time and skill building my system.

For this monent it says me than I will win... maybe in a year or so. YEAR!
I continue purchasing HDD and plotting. As at current speed of growth you have to win a block once a week or so(to have money for the next cluster upgrade). 
If you not win in a week - you have no money. And no chance to catch the winners, who win every minute and can increase theri storage power drammaticaly!
Just 3 days ago calculator was saying than on 50Tb storage I will have win chanse every 14 days. And now it is a 18 days... It not real speed for me to throw such efforts to catch your growing trend...

As I can see the situation, to have a good cluster I need to have something like 150Tb for now, and level it up to 200-250Tb in a month or so. This is not real to a private person. I think only corporations or huge business can afford such quick growth. Not me. 

So? What will you suggest me? I'm no so rich to invest huge money. Will it be better to stop now, before it gets too late? 

And second question. What was the idea of decentralisation? A huge boys control rewards, win every hour and maintain their nodes on received coins. I have to invest my personal money, and having no chance to catch the luck. 

Third question. How soon will be official pool harvesting implemented? 
So I will kill my node and cluster, and start harvesting on the poll for daily reward(like others do on hpoll now)

**Comment by claudio782** [comment]
I agree with you. I suspect that many of us are in the same situation. Apparently the creators of chia have not taken into account the small farmers or have failed to find an effective solution to solve the problem.

**Reply by Kycin1** [reply]
Yes I have thought about this and I think there should have been a cap on space/plots that any one node can have.  Or a increase in size after a certain number of plots.  Like 50 k-32 limit. 50-k33 and so on. So it will gradually get tough at a certain point to swallow the network with 10,000 plots held by one farmer.

**Comment by ghost** [comment]
Hi

Same here, small setup and it looks like the train has left the station .. no chance to catch-up.
Official pool harvesting should be available around 17th of May: https://twitter.com/bramcohen/status/1390407484179763202

**Reply by pluto2021** [reply]
One note on this: the big/small whales have spent significant resource to invest early. It would take some time to gain back that significant initial investment. It might not be straight forward if it is beneficial to invest all new rewards towards new plotting power and more HDD space at an exponential rate. To keep up with the 10-11% daily growth rate, not only any farmer, small or large, has to acquire an exponentially more and more HDD spaces, but also exponentially more and more plotting power. Given the plotting power cost around $30-$40 per plot per day, the cost for the big whales would also grow exponentially.  As such, the fact that the network is growing exponentially most likely due to the number of farmers are growing exponentially, while everyone's plotting power is somewhat limited, and thus the probability of winning for big farmers is also decreasing rapidly. I ran a quick cost model of this rolling investment, and it would not be profitable for anyone trying to catchup the plotting power and HDD space at the current exponential growth rate.  So most likely the network growth would slow down when there would be less people joining on a daily basis when the entry level of investment would not get paid off soon enough, say you might just want to invest in XCH directly, and be more profitable, considering all the purchases that could have been made as a net loss when you would instead buy XCH directly with those investment. If you have done so at the right time, you could have gained 200-300% of your investment today already!

**Reply by ghost** [reply]
Right, for some it‘s just a hobby. Goal might not be to make a living out of it, just to play around and learn how to participate at a global blockchain network.
On the other hand, I think the chain will be less trusted if only a few are in charge. So it should be a vital interest of a global program to aim for a worldwide distribution and stability. Why not also consider IP network / location while selecting the next winner - to avoid single point of failure and 51% attack?
